Rakefile

```
@@ -0,0 +1,126 @@
+require "json"
+
+class Dashboard
+  attr_reader :path
+  attr_accessor :definition
+
+  def initialize(path)
+    @path = path
+  end
+
+  def issues
+    @issues ||= []
+  end
+
+  def validate
+    required_field "metric_keys"
```

```suggestion
    return unless definition
    
    required_field "metric_keys"
```

Do not attempt to validate if the JSON could not be found. (This will be triggered if the `next` is removed -- see comment below)